### ISSUE TITLE
Use on other wikis?

### ISSUE THREAD

**Issue by kj7rrv** [jacob-g/wikimonitor#14]
What is the best way to change this for another wiki? Could I just redefine WIKI_API_URL in conf/logininfo.php?

**Comment by jacob-g** [comment]
Yes, and of course the username and password. You also need to update conf/wikiinfo.php and create all the config pages (all the subpages of [User:WikiMonitor/Configuration](https://en.scratch-wiki.info/wiki/User:WikiMonitor/Configuration)).

**Comment by kj7rrv** [comment]
Okay, thank you! Is it okay to copy those pages from the Scratch wiki (and edit them as needed)? (Does the CC BY-SA license apply to them?)

**Comment by jacob-g** [comment]
Feel free to copy them. Since they aren't really content pages (hosting them on the Wiki is really just for convenience and providing a public ledger for when they're edited) you don't need to bother giving credit for those pages.
